Bookish/Views/Shared/_BookTable.cshtml

```diff
@@ -20,9 +20,8 @@
                 <td>@book.Author</td>
                 <td>@book.Title</td>
                 <td>
-                    @using (Html.BeginForm("ViewCopies", "Home", FormMethod.Post))
+                    @using (Html.BeginForm("ViewCopies", "Home", FormMethod.Get))

```

Comment:
If you just want a button to direct to a specific controller action, you don't need a whole form - you can supply a button input with a `Url.Action`. Take a look at the top answer [here](https://stackoverflow.com/questions/2503923/html-button-calling-an-mvc-controller-and-action-method)